### PR TITLE
fix #75680 imagetwist.com

### LINKED ISSUE #75680: imagetwist.com
### Issue URL (Ads)
 [https://imagetwist.com/cy23c6e66txs/eorltpg6gkx31.jpg](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fimagetwist.com%2Fcy23c6e66txs%2Feorltpg6gkx31.jpg)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#75680
Fixes nasty/NSFW pop-up and the placeholder as described by user.

### CODE REVIEW

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -232,6 +232,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75680
+imagetwist.com#%#//scriptlet("abort-current-inline-script", "document.attachEvent", "initBCPopunder")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:236]: What kind of popups? Not reproduced on my end with US IP, but I can see it on the code. Can you provide an example via screenshot?

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -256,6 +256,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 ||hulkstream.com/jvpop*.js
+imagetwist.com##div[style^="width:300px; height:250px;display: inline-block;"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:259]: Also add this `imagetwist.com#?#.text-center > iframe[name^="spot_id_"]:upward(1)` to the `extended_css.txt`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:259]: Ok. I am not good in fixing ads, but nevermind.

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:236]: @zloyden Check with AdGuard For Windows and disable AdGuard Browser Extension. Then click on the logo, you should see a redirect to `livejasmin.com` and the legitimate website in the next tab.

Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:236]: Weird. let me double-check it.

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:259]: @zloyden How to use write extension CSS rules?

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:236]: Ok, we can add it.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:236]: Okay. @zloyden Is it possible to write HTML rule from here?

Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:236]: The scriptlet is enough here.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:236]: I am just asking. I won't add it.

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/specific.txt:259]: When you see that using basic rules is not enough.
About writing ext-css rules you can read the information [here](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#extended-css-selectors).

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:236]: Yes, it's possible if you mean `$$` rule. It also works here, [but](https://github.com/AdguardTeam/AdguardFilters/pull/75743#discussion_r581914744).

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:236]: @zloyden How to count the characters for min length and max length?

**Comment by Yuki2718** [comment]
Aside from typo, is `magetwist.com#?#.text-center > iframe[name^="spot_id_"]:upward(1)` needed? I don't see difference with `imagetwist.com##div[style^="width:300px; height:250px;display: inline-block;"]`. Extended syntax is not cheap in resource usage so should be avoided if possible.

**Comment by zloyden** [comment]
Rules like this `##div[style^="width:300px; height:250px;display: inline-block;"]` don't last long. `ext-css` rule is needed as an alternative.

**Comment by Yuki2718** [comment]
All right then :)

**Pull request merged**